src/Centeva.ObjectStorage.Azure.Blob/StorageFactoryExtensions.cs

```diff
@@ -5,9 +5,9 @@ public static class StorageFactoryExtensions
     /// <summary>
     /// Register the Azure storage provider.
     /// </summary>
-    public static StorageFactory UseAzureCompatibleStorage(this StorageFactory connectionFactory)
+    public static StorageFactory UseAzureBlobCompatibleStorage(this StorageFactory connectionFactory)
```

Comment:
Why does this have "Compatible"?  Seems like "UseAzureBlobStorage" makes more sense.

Reply:
I copied this file from the UseS3CompatibleStorage. I will remove the "Compatible" from the name